orttraining/orttraining/python/training/onnxblock/_building_blocks.py

```diff
@@ -0,0 +1,146 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# _building_blocks.py
+
+import copy
+import onnx
+
+import onnxruntime.training.onnxblock as onnxblock
+import onnxruntime.training.onnxblock.model_accessor as accessor
+import onnxruntime.training.onnxblock._graph_utils as graph_utils
+
+
+class Sub(onnxblock.Model):
+    """Adds Sub node to an onnx model."""
```

Comment:
why are we adding another layer of abstraction here instead of directly using the onnx helper in the loss class

Reply:
The idea is for users to be able to do something like [this](https://github.com/microsoft/onnxruntime/pull/11520/files#diff-551788247d076b48e3f3e25b5b347fb7c05a04955c013a3da1eb3bce8f76d5e8R64) where the smaller blocks can be stacked on top of each other.
 This way, we can build a frontend to onnx composability.